op_compatibility/first_wave_models.md

```diff
@@ -6,26 +6,35 @@
+| [relu](https://webmachinelearning.github.io/webnn/#api-neuralnetworkcontext-relu) |:heavy_check_mark: |:heavy_check_mark: |:heavy_check_mark: | | :heavy_check_mark: | | [Relu](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Relu) | Lowering to [Max](https://www.tensorflow.org/xla/operation_semantics#element-wise_binary_arithmetic_operations) |
+| [reshape](https://webmachinelearning.github.io/webnn/#api-neuralnetworkcontext-reshape) |:heavy_check_mark: |:heavy_check_mark: | :heavy_check_mark: | | | | [Reshape](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Reshape) | [Reshape](https://www.tensorflow.org/xla/operation_semantics#reshape) |
+| [sigmoid]() | | | | | :heavy_check_mark: | :heavy_check_mark: | [Sigmoid](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Sigmoid) | Lowering to [Tanh](https://www.tensorflow.org/xla/operation_semantics#element-wise_unary_functions) |
+| [softmax](https://webmachinelearning.github.io/webnn/#api-neuralnetworkcontext-softmax) (4) |:heavy_check_mark: |:heavy_check_mark: |:heavy_check_mark: | | | | [Softmax](https://github.com/onnx/onnx/blob/master/docs/Operators.md#softmax) | Lowering to [Add](https://www.tensorflow.org/xla/operation_semantics#element-wise_binary_arithmetic_operations), [Div](https://www.tensorflow.org/xla/operation_semantics#element-wise_binary_arithmetic_operations), [Exp](https://www.tensorflow.org/xla/operation_semantics#element-wise_unary_functions), [Max](https://www.tensorflow.org/xla/operation_semantics#element-wise_binary_arithmetic_operations), [Sub](https://www.tensorflow.org/xla/operation_semantics#element-wise_binary_arithmetic_operations) and [Reduce](https://www.tensorflow.org/xla/operation_semantics#reduce). |
+| [squeeze]() | | | | | | :heavy_check_mark: | [Squeeze](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Squeeze) | Lowering to [Reshape](https://www.tensorflow.org/xla/operation_semantics#reshape) |
+| [tanh]() | | | | | :heavy_check_mark: | | [Tanh](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Tanh) | [Tanh](https://www.tensorflow.org/xla/operation_semantics#element-wise_unary_functions) |
 
 Remarks:
 1. TFLite models don't contain `batchNormalization`. It is fused with other operations during the model conversion.
 2. A depthwise `conv2d` operation is a variant of grouped `conv2d`, used in MobileNet, where the groups = input_channels = output_channels.
 3. TFLite models don't contain `leakyRelu`. It is replaced with other operations during the model conversion.
-4. ONNX models don't contain `softmax`. It is implemented in the post-processing script.
\ No newline at end of file
+4. ONNX models don't contain `softmax`. It is implemented in the post-processing script.
+5. RNNoise implements `gru` as the combination of `sigmoid`, `tanh`, `relu`, `add`, and `mul`.
```

Comment:
Both the update and reset gate of a GRU layer can optionally take an additional bias tensor, which can be more straightforwardly mapped to a `gemm` operation instead of a `matmul` followed by an `add`. Good call on `split` but I would rather use `slice` instead to avoid an operation that produces variadic output tensors. I think `slice` is more generic.

Reply:
Mapping to `gemm` sounds good, as the `transpose` is also used for some cases. I agree with you that `slice` is more generic. So using `slice` also sounds good to me.

Reply:
Your input on `matmul` is valid. We are essentially debating how to implement `gru` in term of other operations. Given that we already marked `gru` as a new operation, whether to implement it as `gemm` or `matmul` seems both equally valid. As for `slice' vs. `split`, I see that both TensorFlow and ONNX support `split` and that `split` is normally used as an inverse operation of `concat` I agree it's more appropriate to define `split`. Note that XLA-HLO will lower `split` down to `slice` anyway. Thanks for the insightful feedback!

Reply:
@huningxin Please look at the latest commit.

Reply:
It looks good. Thanks!